### DISCUSSION TITLE
Read a file from an arbitrary position with `Bun.file` API

### DISCUSSION THREAD

**Discussion by yukulele** [oven-sh/bun#5920] [General]
Hi,

I have a very large file, and I want to read its contents at a specific positions.

* `bunFile.arrayBuffer()` will try to load the full file content in memory
* `bunFile.stream()` forces me to read the contents of the file sequentially (and I have to restart if I want to reach an upstream position).

How can I do this with the bun API?

**Comment by Jarred-Sumner** [comment]
Bun.file is a `Blob`, so you can use `slice` - https://developer.mozilla.org/en-US/docs/Web/API/Blob/slice
